### DISCUSSION TITLE
google pay not working with production settings

### DISCUSSION THREAD

**Discussion by TheoBo** [flutter-stripe/flutter_stripe#1170] [Q&A]
GPay works fine in stripe test mode, but not with production setups.

Using production settings I get the error message:
**The request failed and an unexpected error occurred. Please try again later. [OR_BIBED_07]**

In the instructions I found:

"Google Pay: The plugin can easily act as a payment provider for the Pay plugin, which allows you to seamlessly integrate Google Pay or Apple Pay. All you need to do is add your publishable Stripe key to your profile. payments."

I don't understand what I need to do to "add your publishable distribution key to payments profile".

**Maybe I'm missing some settings at: play.google.com/console?**

Any help would be greatly appreciated

**Comment by jonasbark** [comment]
Try googling OR_BIBED_07 - there are plenty of helpful information out there on that issue
